CONTRIBUTING.md

```diff
@@ -11,3 +18,56 @@ I am truly grateful for all the support developing VSTeam Quickstart Scripts. It
 ### Housekeeping
 
 The VSTeam module runs on macOS, Windows and Linux. Therefore, **casing is very important**.  If your script only works on one platform please describe in the Readme.md file. Also note if this only support a specific version of TFS or requires a minimum API version.
+
+## Contribution Guide
+
+To make sure your scripts are easy to use, please follow these guidelines.
+
+## Files, folders and naming conventions
+
+1. Every script and its associated files must be contained in its own **folder**. Name this folder something that describes what your script does. Usually this naming pattern looks like **level-resource-action** (e.g. 101-git-stale-branches)
+
++ **Required** - Numbering should start at 101. 100 is reserved for things that need to be at the top.
++ **Protip** - Try to keep the name of your script folder short so that it fits inside the GitHub folder name column width.
+
+1. GitHub uses ASCII for ordering files and folder. For consistent ordering **create all files and folders in lowercase**. The only **exception** to this guideline is the **README.md**, that should be in the format **UPPERCASE.lowercase**.
+
+1. Include a **README.md** file that explains how the script works.
+
++ Guidelines on the README.md file below.
+
+1. The script file must be named **run.ps1**.
+
++ Use default values for parameters in the script whenever there is a value that will work for all users. Decorate parameter with `ValidateSet` whenever possible.
```

Comment:
Maybe also add (as said earlier in the script example) to encourage parameters instead of using variables with hardcoded values.